play/src/front/Api/IframeListener.ts

```diff
@@ -430,7 +430,7 @@ class IframeListener {
                     } else if (iframeEvent.type == "showBusinessCard") {
                         requestVisitCardsStore.set(iframeEvent.data.visitCardUrl);
                     } else if (iframeEvent.type == "openModal") {
-                        modalIframeTitlelStore.set(iframeEvent.data.tiltle);
+                        modalIframeTitlelStore.set(iframeEvent.data.title);
```

Comment:
I think there is a typo also on `modalIframeTitlelStore` and on `modalIframeAllowlStore`